Backend/starter_project_mavericks/BlogApp/BlogApp.Application/Features/Authentication/DTO/SignInResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Authentication.DTO
+{
+    public class SignInResponse : SignUpResponse
+    {
+        public string Token { get; set; }
+    }
```

Comment:
on our signin response we can add other properties of user as well if we don't have endpoint that exposes that: this are Id, Username, Firstname ,Lastname and Email of the user.